src/validation/fields.ts

```diff
@@ -75,7 +75,6 @@ export const redirectTo = ENV.AUTH_CLIENT_URL
       .try(
         ...ENV.AUTH_ACCESS_CONTROL_ALLOWED_REDIRECT_URLS.map((value) =>
           Joi.string()
-            .lowercase()
             .regex(new RegExp('^' + value))
```

Comment:
Thanks for this!
That would solve the issue, but on the other hand it would make `AUTH_ACCESS_CONTROL_ALLOWED_REDIRECT_URLS` case sensitive.
Maybe something like this?

```suggestion
            .lowercase()
            .regex(new RegExp('^' + value))
            .raw()
```

Ideally we should add tests. If it's too much asking no worries, I'll take over from there and will do my best to do it by Monday

Reply:
good find, was looking for something like `.raw()` but haven't used `Joi` before so didn't realize that existed. changed in https://github.com/nhost/hasura-auth/pull/279/commits/55ad462398383f3d6d7a889a6d34e4db02684099.

I'd like to contribute more in the future but I don't currently have the time to debug the tests/write them.